apps/web/public/static/locales/en/common.json

```diff
@@ -711,6 +711,8 @@
   "12_hour": "12 hour",
   "24_hour": "24 hour",
   "redirect_success_booking": "Redirect on booking ",
+  "you_are_being_redirected": "You are being redirected to {{ url }} in {{ timeRemaining }} second.",
+  "you_are_being_redirected_plural": "You are being redirected to {{ url }} in {{ timeRemaining }} seconds.",
```

Comment:
oh this works?

Reply:
Yeah. I scratched my head for a long time. I was using _other instead of _plural as per this https://react.i18next.com/latest/trans-component.
Not sure when _plural is used and when _other

Reply:
Instead of pluralizing whole sentences let's try to pluralize single units (seconds in this case) for maximum reusability.

Reply:
> Instead of pluralizing whole sentences let's try to pluralize single units (seconds in this case) for maximum reusability.

I have past experiences with translation where the whole sentence if provided can be given better translation. Not very sure of it's the right approach. 
Would follow what you suggested for now, till I have solid reasoning.

Reply:
@hariombalhara agreed that there will be exceptions but "seconds" I don't think one of them